examples/react-native/src/components/Highlight.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import { Text } from 'react-native';
-import { connectHighlight } from 'react-instantsearch/connectors';
+import { connectHighlight } from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.